templates/containers/index.html

```diff
@@ -13,7 +13,7 @@ <h1>Why is Ubuntu the #1 OS for Containers?</h1>
       <p><a class="js-invoke-modal" href="/containers/contact-us?product=containers-overview">Talk to us about your container strategy&nbsp;&rsaquo;</a></p>
       </div>
       <div class="col-5 u-align--center u-hide--small">
```

Comment:
do you still need the u-align--center class?

Reply:
No, good point!